YearInReview/Model/Reports/MVVM/MainViewModel.cs

```diff
@@ -62,6 +62,9 @@ public MainViewModel(IPlayniteAPI api, ReportManager reportManager)
 						})
 				});
 			}
+
+			YearButtons.FirstOrDefault()?.SwitchYearCommand.Execute(null);
+			ReportButtons.FirstOrDefault()?.DisplayCommand.Execute(null);
```

Comment:
Clever way to trigger the action. 
Maybe add a comment here to explain why we execute the commands of the buttons here, as it may not be so obvious at first look. What do you think? I added a suggestion below;
```suggestion
			// Try to execute the first year and report commands to display the latest review by default
			YearButtons.FirstOrDefault()?.SwitchYearCommand.Execute(null);
			ReportButtons.FirstOrDefault()?.DisplayCommand.Execute(null);
```

Reply:
As per Uncle Bob's "Clean Code" I prefer to avoid comments in my code. Good suggestion about readability - I moved these button clicks into a separate method.

Reply:
That "avoid comments" idea sounds interesting to me ;D 
I was taught to write comments in code, but I get the point that code should be easy to read without needing long explanations.

Adding the method definitely makes it easier to understand. Nice work!